src/plugins/github/fetchGithubRepo.js

```diff
@@ -45,15 +45,28 @@ export default function fetchGithubRepo(
     throw new Error(`Invalid token: ${token}`);
   }
 
-  const query = stringify.body(createQuery(), inlineLayout());
+  const query = createQuery();
   const variables = createVariables(repoOwner, repoName);
   const payload = {query, variables};
-  return postQuery(payload, token);
+  return postQueryExhaustive(
+    (somePayload) => postQuery(somePayload, token),
+    payload
+  ).then((x) => {
+    ensureNoMorePages(x);
+    // TODO: We wrap back up in the "data" object to maintain
+    // compatibility. At some point, let's strip this out and modify
+    // clients of `example-data.json`.
+    return {data: x};
+  });
 }
 
 const GITHUB_GRAPHQL_SERVER = "https://api.github.com/graphql";
 
-function postQuery(payload, token) {
+function postQuery({query, variables}, token) {
+  const payload = {
+    query: stringify.body(query, inlineLayout()),
```

Comment:
I find the re-use of the name "query" for (i assume) the programmatic query template and the laid-out string query slightly confusing, and would appreciate disambiguation

Reply:
Fair. Note that the laid-out string query name is forced by the GitHub API. I considered calling the structured one `body` everywhere, and am still happy to do so. (If there is a reason that I decided not to do that, then I forget the reason.)